### PR TITLE
Paths with Modified Max Accel and Max Speed.  Robot.xml  modified settings

### PR DESCRIPTION
**Pull request by GeorgeSinnott**

### CODE REVIEW

File: src/main/config/robot.xml [hunk 1]
```diff
@@ -3,12 +3,12 @@
 <robot>
 	<!-- pdp canId="0" / -->
 	<pigeon canId="50"
-               rotation="90.0" />
```
Comment (witcpalek) [inline src/main/config/robot.xml:6]: This change should be in practicebot's robot.xml; the pigeon is off by 90 degress on the comp bot

File: src/main/config/robot.xml [hunk 2]
```diff
@@ -141,19 +139,18 @@
                     reverselimitswitch="false"
                     reverselimitswitchopen="true"/>        
 		    <cancoder canId="0"
-                            offset="22.676"/>
+                            offset="71.98242"/>
```
Comment (witcpalek) [inline src/main/config/robot.xml]: looks like practice bot numbers not comp bot numbers

File: src/main/config/robot.xml [hunk 3]
```diff
@@ -249,16 +243,17 @@
...
-                            offset="-156.797"/>
+                            offset="133.242188"/>
         </swervemodule>
 
     </swervechassis>  
  
 <!-- ====================================================
      Mechanisms
      ==================================================== --> 
+     <!--
```
Comment (witcpalek) [inline src/main/config/robot.xml]: Is this the practice bot xml?

**Review by witcpalek — CHANGES_REQUESTED** [review]

**Comment by GeorgeSinnott** [comment]
Yes this is the practice bot xml file.

George Sinnott
Paint Staff Engineer - TIE (TC Automation, SME)
Vehicle Systems – Paint Facilities (JE6C1)
George.sinnott@gm.com
C (248) 941-5406
https://gmweb.gm.com/sites/TCAutomation/SitePages/Home.aspx

[cid:image001.jpg@01D70E76.21E981C0]

**Review by witcpalek — APPROVED** [review]

**Pull request merged**